src/windows/common/tools/README.md

```diff
@@ -0,0 +1,3 @@
+#### Tools included in this folder:
+
+KDbgCtrl (Kernel Debugging Control) -- Used to change the dump configuration without rebooting the system. For more info, see https://docs.microsoft.com/en-us/windows-hardware/drivers/debugger/using-kdbgctrl
```

Comment:
Can you include a HashValue of this exe for security purposes? I want a way for us to validate if a tool has been tempered. https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.utility/get-filehash?view=powershell-7.1

Reply:
Added SHA 256 hash to the file.